portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lorraine Lyu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <ul id="navbar">
+        <li class="nav-button"><a href="index.html" class="nav-link">Home</a></li>
+        <li class="nav-button"><a href="introduction.html" class="nav-link">Introduction</a></li>
+        <li class="nav-button"><a href="projects.html" class="nav-link">Projects</a></li>
+        <li class="nav-button current"><a href="gallery.html" class="nav-link">Gallery</a></li>
+      </ul>
+      <div class="main">
+        <h1>My Image Gallary</h1>
+        <div class="slides">
+          <button class="btn-left" onclick="flip(-1, 1)"><</button>
+          <button class="btn-right" onclick="flip(1, 1)">></button>
+          <img class="pic" src="images/cat0.jpg" alt="not_found.png">
+          <img class="pic" src="images/cat1.jpg" alt="not_found.png">
+          <img class="pic" src="images/cat2.jpg" alt="not_found.png">
+          <img class="pic" src="images/cat3.jpg" alt="not_found.png">
+          <img class="pic" src="images/cat4.jpg" alt="not_found.png">
+        </div>
+      </div>
+        <div class="slides">
+          <button class="btn-left" onclick="flip(-1, 2)"><</button>
+          <button class="btn-right" onclick="flip(1, 2)">></button>
```

Comment:
You said this js function doesn't yet exist right? I would either add it in with this PR or wait to add the click handler. If you choose to add it later, create a bug and add a TODO here.

Reply:
yeah I'll adding it on the next commit.